src/engine/effects/engineeffect.cpp

```diff
@@ -36,6 +55,24 @@ EngineEffect::~EngineEffect() {
     }
 }
 
+EffectState* EngineEffect::createState(const mixxx::AudioParameters& bufferParameters) {
+    if (!m_pProcessor) {
+        return new EffectState(bufferParameters);
+    }
+    return m_pProcessor->createState(bufferParameters);
+}
+
+void EngineEffect::loadStatesForInputChannel(const ChannelHandle& inputChannel,
+    const EffectStatesPointer states) {
+    qDebug() << "EngineEffect::loadStatesForInputChannel" << this << "loading states for input" << inputChannel;
```

Comment:
Be sure to delete qDebug calls from the engine process. They can lead to audio dropouts. 
Alternative you can enable them by the --developer flag only.